platform/darwin/test/MGLDocumentationExampleTests.swift

```diff
@@ -373,7 +373,7 @@ class MGLDocumentationExampleTests: XCTestCase, MGLMapViewDelegate {
         }
         //#-end-example-code
         
-        wait(for: [expectation], timeout: 1)
+        wait(for: [expectation], timeout: 5)
```

Comment:
5&nbsp;seconds is an awfully long time to render such a trivial local style that lacks any external resources. (`MGLStyle.satelliteStreetsStyleURL()` is a slight of hand: a nested class above interposes the method call, returning the one-liner style.)

Reply:
I agree that 5 seconds is a very long time-out, but CI is a very hostile environment for timing-based tests — this can occasionally take up to 3-4 seconds to pass. 😞

Reply:
(Though I’m not ruling out some underlying issue, say, with the performance of the snapshotter.)